portafly/src/i18n/I18nProvider.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import Polyglot from 'node-polyglot'
+
+const polyglot = new Polyglot()
+const I18nContext = React.createContext(polyglot)
```

Comment:
I would call it differently in order to detach from fixed dependencies. Something like `src/localization/LocalizationContext` or something.

Reply:
I don't get it, what is the problem with i18n ?
`i18n` refers to ` internationalization`, not   to a specific dependency as `i18next`

Reply:
You're right, forget about it!